thorlcr/master/thgraphmanager.cpp

```diff
@@ -1457,7 +1457,8 @@ void thorMain(ILogMsgHandler *logHandler, const char *wuid, const char *graphNam
                     thorQueue->connect(false);
                 }
 
-                queryMyNode()->endpoint().getEndpointHostText(instance);
+                queryMyNode()->endpoint().getEndpointIpText(instance);
+                StringBuffer currentWfId; // not filled/not used until recvNextGraph() is called.
```

Comment:
Is this included by mistake?

Reply:
yes, I think must have been when I rebased on to 9.6 (it is introduced later)